1209/ABBA/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: pid
-title: SafeWISE
+title: SafeWISE (HID)
```

Comment:
Do you need a separate PID for HID? Can't you expose both HID and other profiles on the same PID?

Reply:
Yes I need a diferent one. I will copy here the explanation from the previous PR that I submitted:

"Our device SafeWISE, as I have noted in the index.md, uses the TREZOR One firmware (https://github.com/trezor/trezor-mcu).

So the first version of TREZOR One used HID communication and had a single PID for both bootloader and firmware. This is why we requested the PID ABBA months ago.

Later, TREZOR made a big firmware update changing the communication to WebUSB. As you can see in the code, they used 2 new PIDs for the new bootloader and firmware (that they acquired from you). So we need also 2 new PIDs.

In the beginning I thought that I could ask only 1 new PID for the bootloader (ABB9) and use the previous PID (ABBA) for the new firmware. And this is why I requested only the PID ABB9. But as I realized now, after testing the new firmware, it needs to have a different PID than the old firmware because the code decides if it is the HID or WebUSB version based on the firmware PID.

So finally it should be like this:

HID version : PID ABBA for both firmware and bootloader
WebUSB version: PID ABB8 for the bootloader and ABB9 for the firmware"